package.json

```diff
@@ -38,7 +38,8 @@
     "build:docs": "npm run clean:docs && documentation build -g -f html -o ./build/docs ./src/index.js",
     "prepublishOnly": "npm run build:dist && npm run build:docs",
     "release": "np --no-yarn --any-branch && git push https://github.com/terrestris/mapfish-print-manager.git master --tags",
-    "postpublish": "node ./tasks/update-gh-pages.js"
+    "postpublish": "node ./tasks/update-gh-pages.js",
+    "prepare": "npm run build:dist"
```

Comment:
Why is this script needed?

Reply:
In order to have a dist version on the node_modules folder when using a git reference for the version to use on package.json.

https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-scripts